src/components/article/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import './viewArticle.scss';
+import PropTypes from 'prop-types';
+import {
+  TopComponent, LeftSideBar, ArticleContent, RecentArticles, RightSideBar, Footer,
+} from './viewArticleComponents';
+
+import { viewArticle } from '../../redux/actions/viewSingleArticle/viewSingleArticleAction';
+
+
+class ViewArticle extends Component {
+    state = {
+      getArticle: {
+        data: {
+          slug: '', title: '', description: '', body: '', readtime: '', createdAt: '',
+        },
+      },
+    }
+
+    componentDidMount() {
+      const { viewArticle: viewSingleArticle } = this.props;
+      viewSingleArticle('fakeslug2');
+    }
+
+    componentWillReceiveProps(nextProps) {
+      const { getArticle } = this.props;
+      if (nextProps.getArticle !== getArticle) {
+        console.log(nextProps.getArticle);
+        this.setState({ getArticle: nextProps.getArticle });
+      }
+    }
+
+    render() {
+      const { getArticle: { data } } = this.state;
+      console.log(data);
```

Comment:
It would be better if you remove the console.log

Reply:
Fixed